### ISSUE TITLE
Microsoft.DiaSymReader.Native package treats its libraries as content

### ISSUE THREAD

**Issue by piotrpMSFT** [dotnet/symreader#47]
From the nuspec:

```
    <contentFiles>
      <files include="runtimes\win\native\Microsoft.DiaSymReader.Native.x86.dll" buildAction="Content" copyToOutput="true" />
      <files include="runtimes\win\native\Microsoft.DiaSymReader.Native.amd64.dll" buildAction="Content" copyToOutput="true" />
      <files include="runtimes\win\native\Microsoft.DiaSymReader.Native.arm.dll" buildAction="Content" copyToOutput="true" />
    </contentFiles>
```

This has side effects on publish and pack behavior, as exhibited [here](https://github.com/dotnet/cli/issues/5646). If the behavior is by-design, it would be useful to document here to help folks understand why they want this behavior!

**Comment by piotrpMSFT** [comment]
@tmat

**Comment by tmat** [comment]
This wasn't intentional :(. I must have build the package from a wrong branch.

**Comment by tmat** [comment]
If I publish a corrected package 1.4.2 it would get picked up since the dependency is Microsoft.DiaSymReader.Native (>= 1.4.0), correct?

**Comment by tmat** [comment]
Fixed in https://www.nuget.org/packages/Microsoft.DiaSymReader.Native/1.4.2
Workaround is to add a nuget reference to https://www.nuget.org/packages/Microsoft.DiaSymReader.Native/1.4.2
